ui/pages/remote-mode/setup/setup-daily-allowance/remote-mode-setup-daily-allowance.component.tsx

```diff
@@ -88,11 +81,29 @@ export default function RemoteModeSetupDailyAllowance({
   const [dailyLimit, setDailyLimit] = useState<string>('');
   const [isAllowancesExpanded, setIsAllowancesExpanded] =
     useState<boolean>(false);
+  const [selectedAccount, setSelectedAccount] =
+    useState<InternalAccount | null>(null);
+  const [isHardwareAccount, setIsHardwareAccount] = useState<boolean>(false);
```

Comment:
Rename to `isRemoteModeSupported`, `supportsRemoteMode` or similar, since being a hardware account is not what we are checking for.

Reply:
Good call...might address in a follow-up as you suggest just to get this merged asap so it can be used in other tickets being worked on.